egs/fame/s5/run.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+stage=0
+feat_nj=60
+train_nj=60
+decode_nj=20
+corpus=./corpus
+
+# Please check http://www.ru.nl/clst/datasets/ to get the username and password
+
+USERNAME=******
+PASSWORD=******
+
```

Comment:
we generally ask users to download the file manually and place it into a specific location.
That is a little bit less convenient, but more secure, as the credentials are not stored anywhere  (and more importantly, won't get into PR by an accident. Do you feel strongly against that? If not, please do it that way.